### ISSUE TITLE
Remove Windows 1903 SAC from all rancher projects

### ISSUE THREAD

**Issue by nickgerace** [rancher/tasks#164]
### Proposal 

Windows Server 1903 SAC reached EOM on 12/8/2020. We should save on CI time/costs and Dockerhub space by removing 1903 SAC image building/packaging/tagging from future CI runs in all rancher projects

### Upgrade Path

Windows Server 1903 SAC users can drain, cordon, and decommission their nodes. Afterwards, they can add supported SAC or LTSC hosts to the same cluster.

### Context

- **Windows Support**: https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info
- **Windows Servicing Channels**: https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19

**Comment by nickgerace** [comment]
CC: @sirredbeard

**Comment by nickgerace** [comment]
@sirredbeard we should discuss a long term strategy for EOM dockerhub SAC images.

**Comment by sirredbeard** [comment]
@nickgerace Agreed. Do we have any stats on how many customers are still using 1903 SAC?

**Comment by nickgerace** [comment]
Apologies @sirredbeard, just saw this: https://github.com/rancher/tasks/issues/164#issuecomment-814231521
I personally do not. This might be an internal question.
